.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
![missing-req (copy)](https://user-images.githubusercontent.com/93039359/185914025-cdf86147-9cd3-4033-b972-0bf723ab4fea.png)

- [ ] Please add at least 2 inline comments and properly use Github Review Feature :blush: